data/announcements/scheduled.yaml

```
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-10-15T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107
+  endTime: 2022-10-28T18:00:00
+  style: >-
+    background: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(114,114,255) 100%);
```

Where did this specific color come from?
```suggestion
    background: linear-gradient(135deg, rgb(2, 3, 129) 0%, rgb(114, 114, 255) 100%);
```

---

[KubeCon NA 2022 website](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)